features/steps/when.php

```diff
@@ -14,6 +14,12 @@ function invoke_proc( $proc, $mode ) {
 	return $proc->$method();
 }
 
+$steps->When( '/^I start `([^`]+)`$/',
```

Comment:
Could we make this more explicit as "I launch in the background"? "I start" seems too broad to me.